### ISSUE TITLE
Landing page for the product

### ISSUE THREAD

**Issue by Errorname** [Zenika/FAQ#104]
We need a landing page to present the product to potential users

**Comment by Amrithnath** [comment]
Would this be like a page describing the product? @Errorname

**Comment by Errorname** [comment]
Yes, this would be a page that shows the product and allows user to contact us to create their own instance. (In the futur, we want it to be fully automated, kind of like when you create a Slack instance)

**Comment by Amrithnath** [comment]
Do you have a design in mind or a rough idea as to how it should look?

**Comment by Errorname** [comment]
We would like to have a basic version of what Slack(https://slack.com/intl/en-fr/) does: a simple explanation of the product and an input for email address. Followed by some more explanations of the product with images.

<img width="1626" alt="Screenshot 2019-08-02 at 12 07 19" src="https://user-images.githubusercontent.com/6669733/62362836-2a9ba300-b51e-11e9-9b57-2d0fa5c3f969.png">

However, we are open to new ideas

**Comment by Amrithnath** [comment]
Alright I'll get started on a mockup. I think I can share something with you next week on that. After that I'll work on the site.

**Comment by Amrithnath** [comment]
Hey, could you share some product screenshots you would like to see on the frontpage?

**Comment by Errorname** [comment]
Hi, I'll try to send you some screenshots today. In the meantime, you can go to https://demo.faq.team/ and play with the instance for a bit and take some placeholders screenshots if you want?

**Comment by Amrithnath** [comment]
Ok sure Ill do that

**Comment by Amrithnath** [comment]
https://xd.adobe.com/spec/52cc73b1-82c9-43d8-7b14-8d15fb700bb9-24fd/

Hey, here is what I have done so far. Still have to add some product description

**Comment by Errorname** [comment]
That is actually amazing work! I love it! Where did you find the graphics for the 3 people?

**Comment by Amrithnath** [comment]
Thank you, the illustrations can be found at https://undraw.co/

**Comment by frinyvonnick** [comment]
Awesome work @Amrithnath 👏

**Comment by Amrithnath** [comment]
Hey, Isn't the front page at client/public/index.html?

**Comment by Errorname** [comment]
Not really, since the client is a react app, index.html is just the skeleton for the single page app.

Moreover, since FAQ is a multi-tenant app, the home page of `client` is actually the home page of a tenant.

I would like for the `landing` webpage to be separated of the `client`. If you want, you can create a new folder `landing` and put the code there. If you want, you can do the basic html and I'll add it to a new React app after

**Comment by Amrithnath** [comment]
hey @Errorname , I was busy the past two weeks , ill have it ready by this week end

**Comment by frinyvonnick** [comment]
Don't worry @Amrithnath, It's normal to be busy. Take your time! Thank you for your hard work 👍

**Comment by Errorname** [comment]
Hi @Amrithnath, no worries, you are already doing great work and there is no pressure on this :)
